src/main/java/com/google/sps/servlets/GetSessionServlet.java

```diff
@@ -12,6 +23,33 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    throw new Error("Unimplemented");
+    DatastoreClientInterface datastoreClient = new DatastoreClient();
+    String sessionId = 
+        URLDecoder.decode(request.getParameter("session-id"), StandardCharsets.UTF_8);
+    String name = URLDecoder.decode(request.getParameter("name"), StandardCharsets.UTF_8);
+    Optional<SessionInterface> optionalSession; 
+    try {
+      optionalSession = datastoreClient.getSession(sessionId);
+    } catch (Exception e) {
+      response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
+      return;
+    }
+    // Updates the time of the current attendee accessing the GetSessionServlet
+    // for the BackgroundTaskManager.
```

Comment:
Don't mention the BTM. It only further creates more questions and somehow makes the BTM relevant in a place where it doesn't need to be. Just say:
// Update the lastTimePolled for the Attendee making this request. 

Also, please do:
new Attendee(/\*parameterName=\*/blah,...) so it's clear which fields are set